.github/workflows/conan-package.yml

```
@@ -0,0 +1,152 @@
+---
+name: conan-package
+
+on:
+  push:
+    paths:
+      - 'src/**'
+      - 'cmake/**'
+      - 'tests/**'
+      - 'conanfile.py'
+      - 'CMakeLists.txt'
+      - '.github/workflows/conan-package.yml'
+    branches:
+      - main
+      - master
+      - dev
+      - 'release/**'
+      - '!15.*'
+      - '![1-4].[0-9]+'
+    tags:
+      - '![1-4].[0-9]+*'
+      - '!Arachne*'
+      - '!ancient*'
+      - '!jedi*'
+
+  pull_request:
+    paths:
+      - 'src/**'
+      - 'cmake/**'
+      - 'tests/**'
+      - 'conanfile.py'
+      - 'CMakeLists.txt'
+      - '.github/workflows/conan-package.yml'
+    branches:
+      - main
+      - master
+      - dev
+      - 'release/**'
+
+env:
+  CONAN_USER: ${{ secrets.CONAN_USER }}
+  CONAN_PASS: ${{ secrets.CONAN_PASS }}
+
+jobs:
+  get-semver:
+
+    runs-on: ubuntu-20.04
+
+    outputs:
+      semver_full: ${{ steps.git-tool.outputs.fullSemVer }}
+      semver_majorminorpatch: ${{ steps.git-tool.outputs.MajorMinorPatch }}
+      user: ${{ steps.get-conan-broadcast-data.outputs.user }}
+      channel: ${{ steps.get-conan-broadcast-data.outputs.channel }}
+
+    steps:
+      - name: Checkout CuraEngine
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+          fetch-depth: 0
+
+      - name: Install GitVersion
+        uses: gittools/actions/gitversion/setup@v0.9.13
+        with:
+          versionSpec: '5.x'
+
+      - id: git-tool
+        name: GitTools
```

```suggestion
      - name: GitTools
        id: git-tool
```
Nitpicking, but in all other cases the name is first. 


Same also holds for get-conan-broadcast-data below